Tests/XCUITests/HomePageSettingsUITest.swift

```diff
@@ -220,6 +220,29 @@ class HomePageSettingsUITests: BaseTestCase {
 //        waitForExistence(app.cells.staticTexts["Example Domain"], timeout: 3)
     }
 
+    func testRecentlyVisited() {
+        navigator.openURL("mozilla.org")
```

Comment:
We've already have a constant for the URL.
https://github.com/mozilla-mobile/firefox-ios/blob/main/Tests/XCUITests/HomePageSettingsUITest.swift#L7

Reply:
I missed that. I'll change and rebase. Do we have any other changes requested?